PlotInvariantMassFitsLc.C

```diff
@@ -0,0 +1,256 @@
+#if !defined(__CINT__) || defined(__MAKECINT__)
```

Comment:
I would not include this macro in the repository (we typically keep outside or in other repositories the macros for plots, otherwise they proliferate too much)